### ISSUE TITLE
Path to network share is treated as comment

### ISSUE THREAD

**Issue by forki** [tpetricek/FSharp.Formatting#397]
cache //hive/dependencies

See http://fsprojects.github.io/Paket/caches.html#Caching-on-a-network-share

**Comment by LimoWanKenobi** [comment]
Oh, I see. Looks like regular expressions will not be able to handle this kind of cases. I am reviewing what can be done.

**Comment by forki** [comment]
Thanks!
Also I noticed that urls that end in .git are highlighted strangely

**Comment by edhzsz** [comment]
Finally I had some time to work on this.
I have modified the Regex for comments and keywords of Paket and added some more keywords I found in the docs and it works fine.

![image](https://cloud.githubusercontent.com/assets/403823/15567949/9d523358-2329-11e6-86de-f36b51102eaf.png)

![image](https://cloud.githubusercontent.com/assets/403823/15567966/c32bca80-2329-11e6-94c0-a8db79d79b79.png)

There are two caveats although:
* Keywords should have a space after the keyword or after the ":" in order to be recognized and highlighted.
* Comments should have a space after the "//" 

I have created a PR with this changes.

/cc @forki
